dashboard/test/factories/factories.rb

```diff
@@ -360,6 +360,14 @@
         email {''}
         hashed_email {nil}
       end
+
+      trait :without_parental_permission do
+        child_account_compliance_state {''}
+      end
+
+      trait :with_parental_permission do
+        child_account_compliance_state {User::ChildAccountCompliance::PERMISSION_GRANTED}
```

Comment:
My PR does something similar, whoever merges 2nd will have to resolve the conflict.

Reply:
Great minds, n'at. Ok! I'm happy to agree on the order we merge and either do the resolution or pair.

Reply:
You can go ahead and merge first.